include/measurement_kit/common/measure_speed.hpp

```diff
@@ -0,0 +1,54 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef MEASUREMENT_KIT_COMMON_MEASURE_SPEED_HPP
+#define MEASUREMENT_KIT_COMMON_MEASURE_SPEED_HPP
+
+#include <measurement_kit/common/callback.hpp>
+#include <measurement_kit/common/utils.hpp>
+
+namespace mk {
+
+class MeasureSpeed {
```

Comment:
I don't think this class should be here! It's a class used to measure the speed in the NDT test and I thing the correct place would be in the ndt/internal.hpp file, specifically in the Utils section!